environment.yml

```diff
@@ -15,6 +15,8 @@ dependencies:
   - psutil>=5.1.0
   - sphinx
   - sphinx_rtd_theme
+  - dask
```

Comment:
@mlange05 are these, and should these be, compulsory requirements ?

Reply:
Not really, since the use of Dask is not actually part of core Devito. Having it in our "recommended environment" but not in the "hard dependencies" in `setup.py` is thus appropriate, I think.